### PR TITLE
Adds release notes for OMR

### PR DESCRIPTION
**Pull request by stevsmit**
Release notes for OMR. 

For 4.10 and 4.11

Preview: https://stevsmit.github.io/openshift-docs/PROJQUAY-3504/installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-release-notes_installing-mirroring-creating-registry

No QE needed.

Issue: https://issues.redhat.com/browse/PROJQUAY-3931?jql=project%20%3D%20PROJQUAY%20AND%20component%20%3D%20OMR

### CODE REVIEW

File: modules/mirror-registry-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+= Mirror registry for Red Hat OpenShift release notes
+
+The _mirror registry for Red Hat OpenShift_ is a small and streamlined container registry that you can use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+These release notes track the development of the _mirror registry for Red Hat Openshift_ in {product-title}.
+
+For an overview of the _mirror registry for Red Hat OpenShift_, see xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-flags_installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red Hat OpenShift].
+
+
+== Mirror registry for Red Hat OpenShift 1.2.0
```
Comment (jldohmann) [inline modules/mirror-registry-release-notes.adoc:15]: Do these various sections headings need an anchor ID?
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

**Review by jldohmann — APPROVED** [review]
LGTM, left one question below

**Comment by stevsmit** [comment]
cherry-pick enterprise-4.10

**Comment by stevsmit** [comment]
cherry-pick enterprise-4.11

**Pull request merged**